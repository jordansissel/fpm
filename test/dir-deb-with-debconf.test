#!/bin/bash

run() {
  mkdir -p $tmpdir/prefix
  touch $tmpdir/config
  touch $tmpdir/templates

  fpm -s dir -t deb -n testing -a all \
  --deb-config $tmpdir/config \
  --deb-templates $tmpdir/templates \
  $tmpdir/prefix

  file=testing_1.0_all.deb
  dpkg -I $file | grep "\(config\)\|\(templates\)" > $output

  rm $file
}
