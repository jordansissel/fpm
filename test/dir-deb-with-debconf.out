       0 bytes,     0 lines   *  config               
       0 bytes,     0 lines      templates            
